wiki/Tournaments/Mirai/2020/en.md

```diff
@@ -0,0 +1,570 @@
+---
+tags:
+  - Mirai2020
+---
+
+# Mirai. Tournament 2020
+
+![Mirai2020 Logo](img/logo.png)
+
+The **Mirai. Tournament 2020** (**_Mirai2020_**) is a worldwide 2v2 double eliminination osu! tournament hosted by ![][flag_nl] [Mirai Subject](https://osu.ppy.sh/users/5639709). It is the 4th installment of the Mirai. Tournament.
+
+## Tournament Schedule
+
+|                              Event | Timestamp             |
+| ---------------------------------: | :-------------------- |
```

Comment:
```suggestion
| --: | :-- |
```
Please refer to this guideline for table formatting

`:--` - Left Align
`:-:` - Middle Align
`--:` - Right Align

And only add 1 whitespace in the beginning and the end of the structure.
If it's an empty table, tolerance 2 paddings.